### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by cassella**
Fix typos in CPtr.chpl, ChapelRange.chpl, ExternalArray.com,
and chpl-tasks.h

### CODE REVIEW

File: runtime/include/chpl-tasks.h [hunk 1]
```diff
@@ -357,7 +357,7 @@ uint32_t chpl_task_isFixedThread(void) {
 // they started on, this returns true.  Otherwise, it returns false.
 // For example CHPL_TASKS=fifo a task is a thread, so tasks can't
 // migrate.  For CHPL_TASKS=qthreads some schedulers support
-// work-stealing where tasks can be stolen and moved to a diferent
+// work-stealing where tasks can be stolen and moved to a different
```
Comment (ronawho) [inline runtime/include/chpl-tasks.h]: Can you change this to "moved to a different thread then..." while you're here?

**Comment by ronawho** [comment]
Could you rebase to resolve conflicts (and ping me when you do -- for some reason I didn't get a notification for the last force-push)

**Comment by ronawho** [comment]
Thanks Paul!

**Comment by cassella** [comment]
Heh, just realized as I saw the merge email that I should have updated the PR description since ExternalArray isn't in it any more.  And that I'd somehow written ExternalArray.com.

**Pull request merged**